server/server.go

```diff
@@ -0,0 +1,165 @@
+package server
+
+import (
+	"crypto/tls"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/signal"
+	"sync"
+	"syscall"
+	"time"
+
+	"github.com/BurntSushi/toml"
+	"github.com/coniks-sys/coniks-go/merkletree"
+	"github.com/coniks-sys/coniks-go/crypto"
+)
+
+const (
+	Capacity = 1000000
+)
+
+type ServerConfig struct {
+	SigningKeyPath      string  `toml:"signing_key"`
+	Address             string  `toml:"address"` // address:port
+	LoadedHistoryLength int64   `toml:"loaded_history_length"`
+	PoliciesPath        string  `toml:"policies"`
+	TLS                 tlsInfo `toml:"tls"`
+}
+
+type tlsInfo struct {
+	TLSCertPath string `toml:"cert"`
+	TLSKeyPath  string `toml:"key"`
+}
+
+type serverState struct {
+	sync.RWMutex
+	pad *merkletree.PAD
```

Comment:
Yes, I think the changes make sense. 

A couple comments, though. I would merge ``protocol/ops.go`` and ``protocol/directory.go`` since the ops (at least the ones I can think of) are all related to the directory. In the same fashion, I think it makes as much sense to put ``Update()`` in ``ops.go`` as it does to keep it in ``directory.go``. 

My other high-level comment was a question about the ``tbs`` array in ``ConiksDirectory``. Do we want to force the developer to use TBs at this point? Maybe I misunderstood, but I thought we were going to use TBs as part of the reference implementation in ``keyserver``, but not build them into the main protocol implementation.